test/e2e/manifest_test.go

```diff
@@ -49,6 +49,12 @@ var _ = Describe("Podman manifest", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	It("podman manifest inspect", func() {
+		session := podmanTest.Podman([]string{"manifest", "inspect", "docker://library/busybox"})
```

Comment:
Just to be completely sure, could you also try pulling the image specified as it was in the issue.

`podman manifest inspect docker.io/library/busybox`